scripts/budo.js

```diff
@@ -12,29 +12,39 @@ function execCmd (cmd) {
   return p;
 }
 
-var app = budo('./index.js:build/aframe.js', {
+var consts = {
+  NAME: 'AFRAME',
+  ENTRY: './src/index.js',
+  DIST: 'dist/aframe.js',
+  BUILD: 'build/aframe.js',
+  WATCH: 'examples/**/*', // Additional files to watch for LiveReload
```

Comment:
good call with the simplified glob pattern, since the other things will be picked up by browserify